### PR TITLE
PIN-437: Vulnereability scoring feature

### PR DESCRIPTION
**Pull request by pavlix98**

### CODE REVIEW

File: src/BeneficiaryBundle/Model/Vulnerability/Resolver.php [hunk 1]
```diff
@@ -0,0 +1,430 @@
...
+
+        return 0;
+    }
+
+
+    private function computeProductiveAssets(): int
+    {
+        switch (count($this->household->getAssets())) {
+            case 0:
+                return 0; //TODO is it ok?
```
Comment (Troi) [inline src/BeneficiaryBundle/Model/Vulnerability/Resolver.php:373]: Yes, it little weird construction but ok

File: src/BeneficiaryBundle/Model/Vulnerability/ScoringParser.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+    /**
+     * @param array $csv
+     *
+     * @return ScoringValueObject
+     *
+     * @throws ScoringCsvException
+     */
+    public function processCsv(array $csv): object
```
Comment (Troi) [inline src/BeneficiaryBundle/Model/Vulnerability/ScoringParser.php:29]: if you dont need to use it outside and dont test it then it shouldn't be public but protected.

File: src/ProjectBundle/Resources/config/services.yml [hunk 3]
```diff
@@ -36,3 +36,8 @@ services:
         arguments:
             - '@translator'
         public: true
+
+    project.sector_mapper:
+        class: ProjectBundle\Mapper\SectorMapper
+        arguments: []
```
Comment (Troi) [inline src/ProjectBundle/Resources/config/services.yml:42]: it isnt necessary

File: tests/BeneficiaryBundle/Model/Vulnerability/ResolverTest.php [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+    }
+
+
+    public function incomeSpentOnFoodDataProvider()
+    {
+        return [
+            'nullValues' => [null, null, 0],
+            'zero' => [0, 0, 0],
+            'lowVulnerability' => [100, 1000, 1],
+            'highVulnerability' => [800, 1000, 4],
```
Comment (Troi) [inline tests/BeneficiaryBundle/Model/Vulnerability/ResolverTest.php:145]: note: this isnt id, it is only description text, so you can use standard test formatting

**Review by Troi — CHANGES_REQUESTED** [review]
There are some minor requests but it looks good

File: src/ProjectBundle/Resources/config/services.yml (hunk 3 above)
Comment (pavlix98) [reply src/ProjectBundle/Resources/config/services.yml:42]: I think this is your change, I am not sure why it is part of vulnerability pull request.

Comment (Troi) [reply src/ProjectBundle/Resources/config/services.yml:42]: oh, you're right. so it doesnt matter now

**Review by Troi — APPROVED** [review]

**Pull request merged**